default-templates/new-account/index.html

```diff
@@ -16,9 +16,13 @@
   <nav class="navbar navbar-default">
     <div class="container-fluid">
       <div class="navbar-header">
-        <a href="/" class="navbar-brand">{{#if name}}{{name}} &ndash; {{/if}}Solid Home</a>
+        <a href="/" class="navbar-brand">Personal Data Space (POD)</a>
       </div>
       <div class="navbar-right">
+        <a id="logged-href" href="" title="" style="display:none">
+          <span class="glyphicon glyphicon-user"></span>
+        </a>
```

Comment:
Why is this needed?

Reply:
It is for show user icon as href=webId, now you don't see your webId after the Login.